app/src/main/java/com/karumi/screenshot/ui/view/SuperHeroViewHolder.java

```diff
@@ -20,57 +20,60 @@
 import com.squareup.picasso.Picasso;
 
 public class SuperHeroViewHolder extends RecyclerView.ViewHolder {
 
-  private final SuperHeroesPresenter presenter;
-  @Bind(R.id.iv_super_hero_photo) ImageView photoImageView;
-  @Bind(R.id.tv_super_hero_name) TextView nameTextView;
-  @Bind(R.id.iv_avengers_badge) View avengersBadgeView;
+    private final SuperHeroesPresenter presenter;
+    @BindView(R.id.iv_super_hero_photo) ImageView photoImageView;
+    @BindView(R.id.tv_super_hero_name) TextView nameTextView;
+    @BindView(R.id.iv_avengers_badge) View avengersBadgeView;
 
-  public SuperHeroViewHolder(View itemView, SuperHeroesPresenter presenter) {
-    super(itemView);
-    this.presenter = presenter;
-    ButterKnife.bind(this, itemView);
-  }
+    public SuperHeroViewHolder(View itemView, SuperHeroesPresenter presenter) {
+        super(itemView);
+        this.presenter = presenter;
+        ButterKnife.bind(this, itemView);
+    }
 
-  public void render(SuperHero superHero) {
-    hookListeners(superHero);
-    renderSuperHeroPhoto(superHero.getPhoto());
-    renderSuperHeroName(superHero.getName());
-    renderAvengersBadge(superHero.isAvenger());
-  }
+    public void render(SuperHero superHero) {
+        hookListeners(superHero);
+        renderSuperHeroPhoto(superHero.getPhoto());
+        renderSuperHeroName(superHero.getName());
+        renderAvengersBadge(superHero.isAvenger());
+    }
 
-  private void hookListeners(final SuperHero superHero) {
-    itemView.setOnClickListener(new View.OnClickListener() {
-      @Override public void onClick(View v) {
-        presenter.onSuperHeroClicked(superHero);
-      }
-    });
-  }
+    private void hookListeners(final SuperHero superHero) {
+        itemView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                presenter.onSuperHeroClicked(superHero);
+            }
+        });
+    }
 
-  private void renderSuperHeroPhoto(String photo) {
-    if (photo == null) {
-      return;
+    private void renderSuperHeroPhoto(String photo) {
+        if (photo == null) {
+            return;
+        }
+        Picasso.get().load(photo).fit().centerCrop().into(photoImageView);
     }
-    Picasso.with(getContext()).load(photo).fit().centerCrop().into(photoImageView);
-  }
 
-  private void renderSuperHeroName(String name) {
-    nameTextView.setText(name);
-  }
+    private void renderSuperHeroName(String name) {
+        nameTextView.setText(name);
+    }
 
-  private void renderAvengersBadge(boolean isAvenger) {
-    avengersBadgeView.setVisibility(isAvenger ? View.VISIBLE : View.GONE);
-  }
+    private void renderAvengersBadge(boolean isAvenger) {
+        avengersBadgeView.setVisibility(isAvenger ? View.VISIBLE : View.GONE);
+    }
 
-  private Context getContext() {
-    return itemView.getContext();
-  }
+    private Context getContext() {
+        return itemView.getContext();
+    }
```

Comment:
All the changes in this file are not needed. If you configure your IDE to use the expected checkstyle all these changes will disappear.